opal/mca/common/ofi/common_ofi.c

```diff
@@ -87,19 +91,30 @@ OPAL_DECLSPEC int opal_common_ofi_init(void)
 {
     int ret;
 
+    /*
+     * Global refcnt and boolean are used to ensure we only initialize once
+     */
     opal_common_ofi_init_ref_cnt++;
     if (opal_common_ofi_initialized) {
         return OPAL_SUCCESS;
     }
 #if OPAL_OFI_IMPORT_MONITOR_SUPPORT
 
+    /*
+     * Checks for an available memory monitor. If none exist, we do not
+     * initialize the import monitor.
+     */
     mca_base_framework_open(&opal_memory_base_framework, 0);
     if ((OPAL_MEMORY_FREE_SUPPORT | OPAL_MEMORY_MUNMAP_SUPPORT)
         != (((OPAL_MEMORY_FREE_SUPPORT | OPAL_MEMORY_MUNMAP_SUPPORT))
         & opal_mem_hooks_support_level())) {
         return OPAL_SUCCESS;
     }
 
+    /*
+     * This cache object doesn't do much, but is necessary for the API to work.
```

Comment:
this doesn't really help the reader.  Bigger questions would be "Why 1.13" and "why "mr_cache"".  Those are the things you should comment on.